spec/models/alchemy/picture_spec.rb

```diff
@@ -446,27 +354,6 @@ module Alchemy
       end
     end
 
-    context "navigating records" do
-      let!(:picture1) { create(:alchemy_picture, name: "abc") }
-      let!(:picture2) { create(:alchemy_picture, name: "def") }
-
-      describe "#previous" do
-        subject { picture2.previous }
-
-        it "returns the previous record by name" do
-          is_expected.to eq(picture1)
-        end
-      end
-
-      describe "#next" do
-        subject { picture1.next }
-
-        it "returns the next record by name" do
-          is_expected.to eq(picture2)
-        end
-      end
-    end
-
```

Comment:
Do we still have convered this in the controller specs?

Reply:
Yeah, we actually do. It covers even a bit more than this here because it also tests the variable assignment for the views: https://github.com/AlchemyCMS/alchemy_cms/blob/main/spec/controllers/alchemy/admin/pictures_controller_spec.rb#L179-L196

Reply:
Great!